databricks/koalas/series.py

```diff
@@ -2883,12 +2883,12 @@ def transform_batch(self, func, *args, **kwargs) -> "ks.Series":
 
         Examples
         --------
-        >>> df = ks.DataFrame([(1, 2), (3, 4), (5, 6)], columns=['A', 'B'])
+        >>> df = ks.DataFrame([(1, -2), (3, -4), (5, -6)], columns=['A', 'B'])
```

Comment:
Let's don't try to fix the existing tests as possible.

I think maybe you can create another `Series` for test `np.abs`, or simply just test with other `np.ufunc` just like the below if there is no special reason to should test the `np.abs`.

```python
>>> df.A.transform_batch(np.add, 10)
0    11
1    13
2    15
Name: A, dtype: int64
```